graylog2-web-interface/src/views/logic/aggregationbuilder/AggregationWidget.ts

```diff
@@ -19,49 +19,51 @@ import { Map } from 'immutable';
 import isDeepEqual from 'stores/isDeepEqual';
 import isEqualForSearch from 'views/stores/isEqualForSearch';
 import type { QueryString, TimeRange } from 'views/logic/queries/Query';
+import type { FiltersType } from 'views/types';
 
 import AggregationWidgetConfig from './AggregationWidgetConfig';
 
 import Widget from '../widgets/Widget';
 
 export default class AggregationWidget extends Widget {
-  constructor(id: string, config: AggregationWidgetConfig, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = []) {
-    super(id, AggregationWidget.type, config, filter, timerange, query, streams);
+  // eslint-disable-next-line default-param-last
+  constructor(id: string, config: AggregationWidgetConfig, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = [], filters?: FiltersType) {
```

Comment:
Afaics we do not rely on the default value for the `streams` parameter, so we can just remove it. This way we do not need to suppress `default-param-last` for the `filters` parameter.